src/s3ql/backends/gs.py

```diff
@@ -53,6 +53,14 @@ def __init__(self, options):
         if self.use_oauth2:
             self.hdr_prefix = 'x-goog-'
 
+        self.google_app_credentials = None
+        if self.password == '':
+            log.info('Password empty, using google application default account')
+            # Don't import at top level for backwards compatibility
+            import google.auth
```

Comment:
I'd rather have a condition import at the top and only use the module if it's available. Having it crash with an ImportError is ugly.